cmd/litcli/proxy.go

```diff
@@ -22,6 +27,29 @@ var litCommands = []cli.Command{
 		Category: "LiT",
 		Action:   getInfo,
 	},
+	{
+		Name: "bakesupermacaroon",
+		Usage: "Bake a new super macaroon with all of LiT's active " +
+			"permissions.",
+		Category: "LiT",
+		Action:   bakeSuperMacaroon,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name: "suffix",
```

Comment:
nit: I think in this context the flag should be called "root_key_suffix", otherwise it might be confusing.